Sources/Logging/Locks.swift

```diff
@@ -26,12 +26,18 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if canImport(WASILibc)
+// No locking on WASILibc
+#else
+
 #if os(macOS) || os(iOS) || os(tvOS) || os(watchOS)
 import Darwin
 #elseif os(Windows)
 import WinSDK
-#else
+#elseif canImport(Glibc)
```

Comment:
This should make `strftime` and `stdout`/`stderr` available.

```suggestion
#elseif os(WASI)
import WASILibc
#elseif canImport(Glibc)
```